src/js/media-time-range.js

```diff
@@ -51,15 +51,27 @@ template.innerHTML = `
     }
 
     [part~="preview-box"] {
-      transition: visibility .25s, opacity .25s;
```

Comment:
these changes below add the possibility to change the time range preview time/thumb transition, specifically for the start delay which Matt called out. it's a better experience if it has a small delay on mouse hover instead of showing up right away

Reply:
suggestion(non-blocking): I think this is fine for now, but I think these at best have a non-trivial learning curve and at worst folks may break things when trying to use them (especially `--media-preview-transition-property`). We may want to do a followup to figure out a simpler way of modeling this stuff.